### PR TITLE
Improve import instructions

### PR DESCRIPTION
**Pull request by trishankatdatadog**

### CODE REVIEW

File: docs/optional.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ You don't need to do anything extra if you have not set up GPG and SSH to your u
 
 Otherwise, you need to:
 
-1. Get a copy of your Yubikey GPG public key, this should have been backed up in your password manager
-2. Get the Yubikey GPG key ID by running `gpg --list-keys`, in the following example the key ID is `4E09860E71D948019BD426D5D099A306DBECDF1B`  
+1. Get a copy of your Yubikey GPG public key, this should have been backed up in your password manager.
+2. Write your copy of your GPG public key stored in your password manager to disk (e.g., to `/path/to/pubkey.asc`).
+3. Import your public key to GPG: `gpg --import /path/to/pubkey.asc`
```
Comment (daisukixci) [inline docs/optional.md:17]: The import script deals with the import of the public key https://github.com/DataDog/yubikey/blob/master/import.sh#L72

Comment (trishankatdatadog) [reply docs/optional.md:17]: But where do you get the GGP key ID in (4) without it? There's a bit of circularity here...

Comment (daisukixci) [reply docs/optional.md:17]: You are supposed to have it from previous laptop it should even be the filename of the pub key if they used this repo

Comment (trishankatdatadog) [reply docs/optional.md:17]: This is all just not clear right now, I'm afraid. How should we improve it?

Comment (trishankatdatadog) [reply docs/optional.md:17]: Right, so would you please propose a suggestion, or let's close this issue as WONTFIX? Thanks!

Comment (daisukixci) [reply docs/optional.md]: done

**Review by daisukixci — APPROVED** [review]

**Pull request merged**